src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -60,16 +60,17 @@
                     <div class="m-2">
                         @if (Model.IsResolved)
                         {
-                            <form method="post" asp-controller="Comments" asp-action="Unresolve" asp-route-reviewId="@Model.ReviewId">
+                            <form method="post" class="inline" asp-controller="Comments" asp-action="Unresolve" asp-route-reviewId="@Model.ReviewId">
                                 <button data-post-update="comments" type="submit" name="submit" value="Submit" class="btn btn-outline-secondary">Unresolve</button>
                             </form>
                         }
                         else
                         {
-                            <form method="post" asp-controller="Comments" asp-action="Resolve"  asp-route-reviewId="@Model.ReviewId">
+                            <form method="post" class="inline" asp-controller="Comments" asp-action="Resolve"  asp-route-reviewId="@Model.ReviewId">
```

Comment:
Why was this required?

Reply:
I added this so the buttons would sit next to each other.